### ISSUE TITLE
Raw transaction do not work in 2.0.11 version

### ISSUE THREAD

**Issue by Ivanov-Yuriy** [orientechnologies/orientdb#4485]
Hi I'm using node.js and oriento.

After database upgrade to 2.0.11
I have not worked raw transaction
oriento's tests have not worked too

should execute a complex transaction, using a raw query,
should execute a complex transaction, using the query builder
should execute a complex transaction, using the query builder for let statements

Is this bug or orientDB does not support raw transaction anymore?

p.s. orientjs tests have not worked too

**Comment by lvca** [comment]
Could you provide more information about any error, logs or way to reproduce the problem?

**Comment by Ivanov-Yuriy** [comment]
sorry I have no time now, I will write tomorrow necessarily

**Comment by Ivanov-Yuriy** [comment]
I have code

var Oriento =  require('oriento')
var server = Oriento({
    host: 'localhost',
    port: 2424,
    username: 'root',
    password: 'password',
    transport: 'binary'
});

var db = server.use({
    name:       'test',
    username:   'admin',
    password:   'admin'
});

**Comment by maggiolo00** [comment]
hi @Ivanov-Yuriy 

do you still need help on this?

**Comment by Ivanov-Yuriy** [comment]
Hi @maggiolo00 
I'm sorry to keep you waiting
I would like to this bug was fixed

**Comment by maggiolo00** [comment]
@Ivanov-Yuriy 

do you still have this issue with orientjs 2.2.x and OrientDB 2.2.x?

**Comment by Ivanov-Yuriy** [comment]
@maggiolo00 
Yes, I use orientjs 2.2.2 and OrientDB 2.2.11 now

**Comment by maggiolo00** [comment]
@Ivanov-Yuriy 

you can use array notation

```
    var query = 'begin\n' +
      'let test = select from OUser\n' +
      'let count = select COUNT(*) from OUser\n' +
      'let meta = select * from ( select expand(classes) from metadata:schema ) WHERE name = \'OUser\'\n' +
      'let rez = [$test, $count, $meta] \n' +
      'commit retry 100\n' +
      'return $rez';
```

or Object notation

```
 var query = 'begin\n' +
      'let test = select from OUser\n' +
      'let count = select COUNT(*) from OUser\n' +
      'let meta = select * from ( select expand(classes) from metadata:schema ) WHERE name = \'OUser\'\n' +
      'let rez = { test:  $test, count :  $count, meta: $meta  }\n' +
      'commit retry 100\n' +
      'return $rez';
```

**Comment by maggiolo00** [comment]
@Ivanov-Yuriy 

i've added extra test for this

https://github.com/orientechnologies/orientjs/commit/6546191274c8e4d2456710e4623431d63e261f17

**Comment by Ivanov-Yuriy** [comment]
it'll be fine
I'm waiting changes
thank you

**Comment by maggiolo00** [comment]
@Ivanov-Yuriy 

i've released the new 2.2.3 version of OrientJS. 
It should over the example above

Thanks
